### ISSUE TITLE
Cannot load collections

### ISSUE THREAD

**Issue by MatteoGioioso** [a85/POSTMan-Chrome-Extension#10155]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Describe the Issue

It has been a couple of days that both the webapp and the desktop app are not working for me.

I will show you what my main screen looks like:

![image](https://user-images.githubusercontent.com/43592430/126962569-6be1f5c7-6570-4719-b5df-8e5001ad5d10.png)

Unfortunately I am not able to tell you more since that's basically it.
I have inspected the console and the network tab, but there is nothing suspicious, apart for a `CONNECTION_REFUSED`to `localhost:10533/knockknock` which should be the desktop agent if I am not mistaken.

Let me know if I can give you more information.

### What I have tried so far

- Restart my laptop
- Clear the site data
- logout and login again
- try different browser: Chrome, Firefox
- try different internet connection 

### Steps To Reproduce

1. Login with email and password

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/43592430/126962569-6be1f5c7-6570-4719-b5df-8e5001ad5d10.png)

### Environment Information

```markdown
- Operating System: Ubuntu 18.04
- Platform Type: Web and desktop
- Postman Version: latest
```

### Additional Context?

- I do not use any VPN or proxy

**Comment by postman-staff** [comment]
Hey @MatteoGioioso,

Really sorry to hear you are facing an issue with loading collections in your workspace.

Also thanks for giving us the details and clarifying you are not behind proxy or VPN and even tried connecting to another network to rule out network issues.

- Are you seeing this issue only for a specific workspace or it happens to all of your workspaces (even a workspace with only a small number of requests)?
- Can you please try our Canary version and see if that helps? https://www.postman.com/downloads/canary/
- Also, if possible, would you please try accessing the workspace on another device to rule out device dependency?

Thanks for your cooperation on this.
I am looking forward to hearing back from you!

via @taehoshino | Postman Support

**Comment by MatteoGioioso** [comment]
Ok, I have partially fixed the issue pointing my DNS to 1.1.1.1 (essentially using WARP), postman web is still slow, but at least now it is loading.

Thanks, I guess you can close this issue.

**Comment by postman-staff** [comment]
Hey @MatteoGioioso,

Thanks for letting us know the issue has been resolved!

Please do feel free to reach out if there is anything we can help!

via @taehoshino | Postman Support
